store/src/main/java/com/nytimes/android/external/store/base/impl/BarCode.java

```diff
@@ -24,7 +28,7 @@ public BarCode(@Nonnull String type, @Nonnull String key) {
 
     @Nonnull
     public static BarCode empty() {
-        return new BarCode("", "");
```

Comment:
BarCode is immutable, better create a static instancd of the empty barcode and return that.